README.md

```diff
@@ -46,3 +46,32 @@ var myCloseHandler = function() {
   this.setState({showModal: false});
 }
 ```
+
+## Contributing
+
+### When to add a component
+
+* Rule of 3ish - Copy and pasting it more than once? / it's being used a few places? Yes, componentize it!
+* It's in our desired components list
+
+#### Desired components
+
+**Definitely Want**
+* ~~Buttons~~ (done)
+* ~~Modals~~ (done)
+* Form elements - input, select, radio, checkbox, textarea
+* Flash messages
+* Alerts - sub type of Modal
+* Tables
+
+**Someday Want**
+
+* Header
+* Navbar
+* Breadcrumbs
+* Pagination
+
+### How to add a component
+* Add a new folder to `/src` with your component
+* Create an example of how to use your component in code
+* Add a working/live example of your component to the docs
```

Comment:
* Export your component in `index.js`
* Add a test in /test named `[component]_test.js`